deploy/charts/rook-ceph/templates/clusterrole.yaml

```
@@ -21,6 +21,12 @@ rules:
   - apiGroups: ["admissionregistration.k8s.io"]
     resources: ["validatingwebhookconfigurations"]
     verbs: ["create", "get", "delete", "update"]
+  - apiGroups: ["csiaddons.openshift.io"]
+    resources: ["networkfences"]
+    verbs: ["create", "get", "update","delete","watch","list"]
```

nit: consistent spacing
```suggestion
    verbs: ["create", "get", "update", "delete", "watch", "list"]
```